redbot/cogs/warnings/warnings.py

```diff
@@ -76,6 +84,49 @@ def __init__(self, bot: Red):
         else:
             await ctx.send(_("Custom reasons have been disabled."))
 
+    @warningset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context):
+        """Toggle warns being sent to the users DM's"""
+        guild = ctx.guild
+        toggle = await self.config.guild(guild).toggle_dm()
+        await self.config.guild(guild).toggle_dm.set(not toggle)
+        toggle = await self.config.guild(ctx.guild).toggle_dm()
+        if toggle == True:
+            await ctx.send("Warns will now be sent to users DM's")
+        else:
+            await ctx.send("Warns will not be sent to users DM's")
+
+    @warningset.command()
+    @commands.guild_only()
+    async def warnchannel(self, ctx: commands.Context, channel=None):
+        """Set the channel that warns get set to"""
+        guild = ctx.guild
+        try:
+            if channel == None:
+                channel = ctx.channel
+                await self.config.guild(guild).warn_channel.set(channel.id)
+                await ctx.send(f"The warn channel has been set to {channel.mention}")
+            else:
+                channel = await (TextChannelConverter()).convert(ctx, channel)
+                await self.config.guild(guild).warn_channel.set(channel.id)
+                await ctx.send(f"The warn channel has been set to {channel.mention}")
+        except commands.BadArgument:
+            return await ctx.send("Couldn't find that channel.")
+
+    @warningset.command()
+    @commands.guild_only()
+    async def togglechannel(self, ctx: commands.Context):
+        """Toggle if the warn should be sent to a channel"""
+        guild = ctx.guild
+        toggle = await self.config.guild(guild).toggle_channel()
+        await self.config.guild(guild).toggle_channel.set(not toggle)
+        toggle = await self.config.guild(ctx.guild).toggle_channel()
```

Comment:
Same thing with above, you can use `not toggle` instead of calling Config again.